### PR TITLE
[Documentation] Small fixes to "Your First Move Module"

### PR DESCRIPTION
**Pull request by JoshLind**
## Motivation

This PR makes some small fixes to the developer documentation: Your First Move Module, found here: https://aptos.dev/tutorials/your-first-move-module

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

Manually inspected all of the change sites.

## Related PRs

None.

### CODE REVIEW

File: developer-docs-site/docs/tutorials/first-move-module.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ import TabItem from '@theme/TabItem';
 
 # Your first Move Module
 
-This tutorial details how to write, compile, test, publish, and interact with Move Modules on the Aptos Blockchain. The steps to doing so:
+This tutorial details how to write, compile, test, publish and interact with Move Modules on the Aptos Blockchain. The steps to doing this are:
 
-* Write, compile, and test the Move Module
-* Publish the Move Module to the Aptos Blockchain
-* Initialize and interact with resources of the Move Module
+1. Write, compile and test the Move Module
```
Comment (davidiw) [inline developer-docs-site/docs/tutorials/first-move-module.md:14]: no, I am sorry, I cannot accept this, sincerely. the oxford comma is mandatory.

**Review by davidiw — CHANGES_REQUESTED** [review]
good besides ... ","

File: developer-docs-site/docs/tutorials/first-move-module.md [hunk 2]
```diff
@@ -9,13 +9,13 @@ import TabItem from '@theme/TabItem';
 
 # Your first Move Module
 
-This tutorial details how to write, compile, test, publish, and interact with Move Modules on the Aptos Blockchain. The steps to doing so:
+This tutorial details how to write, compile, test, publish and interact with Move Modules on the Aptos Blockchain. The steps to doing this are:
```
Comment (gregnazario) [inline developer-docs-site/docs/tutorials/first-move-module.md:12]: Let's cut down the words.  `The steps are:`

File: developer-docs-site/docs/tutorials/first-move-module.md [hunk 3]
```diff
@@ -9,13 +9,13 @@ import TabItem from '@theme/TabItem';
...
 
-* Write, compile, and test the Move Module
-* Publish the Move Module to the Aptos Blockchain
-* Initialize and interact with resources of the Move Module
+1. Write, compile and test the Move Module
+2. Publish the Move Module to the Aptos Blockchain
+3. Initialize and interact with resources of the Move Module
 
-This tutorial builds on "[Your first transaction](/tutorials/your-first-transaction)" and borrows that code as a library for this example. The following tutorial contains example code that can be downloaded in its entirety below:
+This tutorial builds on [Your first transaction](/tutorials/your-first-transaction) and borrows the code as a library for this example. The following tutorial contains example code that can be downloaded in its entirety, below:
```
Comment (gregnazario) [inline developer-docs-site/docs/tutorials/first-move-module.md:18]: I don't think the comma after entirety belongs?

File: developer-docs-site/docs/tutorials/first-move-module.md [hunk 4]
```diff
@@ -88,7 +88,7 @@ module HelloBlockchain::Message {
 }
 ```
 
-In the previous code, the two important sections are the struct `MessageHolder` and the function `set_message`. `set_message` is a `script` function allowing it to be called directly by transactions. Upon calling it, the function will determine if the current account has a `MessageHolder` resource and creates and stores the `message` it if it does not exist. If the resource exists, the `message` in the `MessageHolder` is overwritten.
+In this code, the two important sections are the struct `MessageHolder` and the function `set_message`. `set_message` is a `script` function allowing it to be called directly by transactions. Upon calling it, the function will determine if the current account has a `MessageHolder` resource and creates and stores the `message` it if it does not exist. If the resource exists, the `message` in the `MessageHolder` is overwritten.
```
Comment (gregnazario) [inline developer-docs-site/docs/tutorials/first-move-module.md:91]: This usually refers to below?  Should we put it above the code?

File: developer-docs-site/docs/tutorials/first-move-module.md (hunk 1 above)
Comment (JoshLind) [reply developer-docs-site/docs/tutorials/first-move-module.md]: Incredible! 😄

File: developer-docs-site/docs/tutorials/first-move-module.md (hunk 3 above)
Comment (JoshLind) [reply developer-docs-site/docs/tutorials/first-move-module.md]: Yeah, probably doesn't matter that much. I've removed it 😄

File: developer-docs-site/docs/tutorials/first-move-module.md (hunk 4 above)
Comment (JoshLind) [reply developer-docs-site/docs/tutorials/first-move-module.md]: I've left it where it is and updated the text to say: "In the code above, ..."

**Review by davidiw — APPROVED** [review]

File: developer-docs-site/docs/tutorials/first-move-module.md [hunk 5]
```diff
@@ -88,7 +88,7 @@ module HelloBlockchain::Message {
 }
 ```
 
-In the previous code, the two important sections are the struct `MessageHolder` and the function `set_message`. `set_message` is a `script` function allowing it to be called directly by transactions. Upon calling it, the function will determine if the current account has a `MessageHolder` resource and creates and stores the `message` it if it does not exist. If the resource exists, the `message` in the `MessageHolder` is overwritten.
+In the code above, the two important sections are the struct `MessageHolder` and the function `set_message`. `set_message` is a `script` function allowing it to be called directly by transactions. Upon calling it, the function will determine if the current account has a `MessageHolder` resource and creates and stores the `message` it if it does not exist. If the resource exists, the `message` in the `MessageHolder` is overwritten.
```
Comment (CapCap) [inline developer-docs-site/docs/tutorials/first-move-module.md:91]: “It if it” -> “if it”

**Review by CapCap — APPROVED** [review]
One tiny change otherwise LGTM, thanks!

**Comment by JoshLind** [comment]
Thanks, all!

/land

**Pull request merged**